### ISSUE TITLE
Claper accessibility with screen reader

### ISSUE THREAD

**Issue by bpisch** [ClaperCo/Claper#158]
Hello,
Recently we started to organise events for people with visual impairment (myself included) where the need appeared for an accessible ARS system. I'm doing tests on Claper for a few days now and I like that I can self-host it and it is rich in functionality, but I have identified some key areas which are not accessible for a screen reader software, and I'm planning to improve these. Just a few such examples:
- a lot of buttons don't have a label readable by the screen reader, so they are announced as "unlabelled"
- some information/features are available with hidden checkboxes where an SVG image is marking the position for clicking the checkbox, but since the checkbox is hidden it's not announced
- some elements on the page aren't accessible by keyboard, or just with advanced cursor movement tricks
- on the organiser view ("manage") the actions of the audience aren't properly announced

I have some experience in web accessibility, so I plan to solve these problems by forking. My aim is to solve these problems by at most minimal modification to the current interface, but ideally I would prefer if I wouldn't need to touch the visual look of the application at all. So I plan to solve everything using aria-attributes on already-existing elements, and with special CSS attributes which make information available only for the screen reader, but doesn't bother people who already get this information visually.
For this project, I would like to stick to WCAG2 standard as much as I can so that there is an official standard backing all the changes I make.
My question is, is there any interest, comment, question about these changes? If there is interest in accessibility for Claper, I can open a pull request when the work is done. Or if anyone has any comments, questions, I'm open for discussing them.

**Comment by bpisch** [comment]
Hello,
Alright, then I will contribute my changes back to dev. There is another question however: should I integrate the newly-added aria-labels into gettext?
I can't do more languages than English and Hungarian though.

**Comment by alxlion** [comment]
Yes, the best thing to do here is to use gettext. Don't worry about other languages, the community will help out 😉 

(I didn't forget your contribution for Hungarian #160)
